pkgs/unified_analytics/lib/src/constants.dart

```diff
@@ -92,18 +92,19 @@ const String kSessionFileName = 'dart-flutter-telemetry-session.json';
 
 /// The message that should be shown to the user.
 const String kToolsMessage = '''
-The [tool name] uses Google Analytics to report usage and diagnostic data
-along with package dependencies, and crash reporting to send basic crash reports.
-This data is used to help improve the Dart platform, Flutter framework, and related tools.
-
-Telemetry is not sent on the very first run.
-To disable reporting of telemetry, run this terminal command:
-
-[dart|flutter] --disable-analytics.
-If you opt out of telemetry, an opt-out event will be sent,
-and then no further information will be sent.
-This data is collected in accordance with the
-Google Privacy Policy (https://policies.google.com/privacy).
+The [tool name] uses Google Analytics to report usage and diagnostic
```

Comment:
As a very minor nit, you may consider using `{{ variable }}` as your templating pattern. It's not uncommon due to things like mustache.